doc/design-qemu-blockdev.rst

```diff
@@ -0,0 +1,175 @@
+==========================
+Ganeti daemons refactoring
```

Comment:
I think this heading needs changing (copy-pasted from design-daemons.rst ?)

Reply:
Fixed, thanks :-)